<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadDialog</class>
 <widget class="QDialog" name="DownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>144</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Download</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Bytes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ETA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="bytesLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--- of ---</string>
   </property>
  </widget>
  <widget class="QLabel" name="etaLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--- h --- m</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--- mbit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
